api/src/opentrons/protocol_api_experimental/pipette_context.py

```diff
@@ -211,8 +211,24 @@ def touch_tip(  # noqa: D102
         v_offset: float = -1.0,
         speed: float = 60.0,
     ) -> PipetteContext:
-        # TODO: https://github.com/Opentrons/opentrons/issues/9526
-        raise NotImplementedError()
+
+        if location is None:
+            raise NotImplementedError("Not including location not yet supported in PE.")
+        if radius != 1.0:
+            raise NotImplementedError("Radius adjustment not yet supported in PE.")
+        if speed != 60.0:
+            raise NotImplementedError("Speed parameter not yet supported in PE.")
+
+        self._engine_client.touch_tip(
+            pipette_id=self._pipette_id,
+            labware_id=location.parent.labware_id,
+            well_name=location.well_name,
+            well_location=WellLocation(
+                origin=WellOrigin.BOTTOM,
```

Comment:
Oops, missed this in my first review. Definitely needs to be 1mm down from the **top** of the well, not 1mm down from the bottom of the well, which would not be pretty.

```suggestion
                origin=WellOrigin.TOP,
```

You can see this in the PAPIv2 implementation of this logic, using its slightly different geometry syntax:

https://github.com/Opentrons/opentrons/blob/0e29be3c5775651290a3af3471a4bf7bc8f217c3/api/src/opentrons/protocol_api/instrument_context.py#L547-L549